lib/membrane_rtc_engine/endpoints/webrtc/connection_prober.ex

```diff
@@ -1,91 +1,175 @@
 defmodule Membrane.RTC.Engine.Endpoint.WebRTC.ConnectionProber do
   @moduledoc false
 
+  @behaviour Membrane.RTC.Engine.Endpoint.WebRTC.ConnectionAllocator
```

Comment:
put this after aliases

Reply:
Can't do that, credo enforces that behaviors are before aliases

Reply:
And it seems that credo config isn't out of date. Sorry, those are the rules

Reply:
:cry: